scipy/stats/_multivariate.py

```diff
@@ -700,22 +700,32 @@ class matrix_normal_gen(multi_rv_generic):
     -----
     %(_matnorm_doc_callparams_note)s
 
-    The covariance matrices `rowcov` and `colcov` must be (symmetric) positive
-    definite matrices. If the samples in `X` are :math:`m \times n`, then
-    `rowcov` must be :math:`m \times m` and `colcov` must be
-    :math:`n \times n`. `mean` must be the same shape as `X`.
+    The covariance matrices specified by `rowcov` and `colcov` must be
+    (symmetric) positive definite. If the samples in `X` are
+    :math:`m \times n`, then `rowcov` must be :math:`m \times m` and
+    `colcov` must be :math:`n \times n`. `mean` must be the same shape as `X`.
 
     The probability density function for `matrix_normal` is
 
     .. math::
 
         f(X) = (2 \pi)^{-\frac{mn}{2}}|U|^{-\frac{n}{2}} |V|^{-\frac{m}{2}}
-               \exp\left( -\frac{1}{2} \trace\left[ U^{-1} (X-M) V^{-1}
+               \exp\left( -\frac{1}{2} \mathrm{Tr}\left[ U^{-1} (X-M) V^{-1}
                (X-M)^T \right] \right),
 
     where :math:`M` is the mean, :math:`U` the among-row covariance matrix,
     :math:`V` the among-column covariance matrix.
 
+    The `allow_singular` behaviour of the `multivariate_normal`
+    distribution is not currently supported. Covariance matrices must be
+    full rank.
+
+    The `matrix_normal` distribution is closely related to the
+    `multivariate_normal` distribution. Specifically, :math:`\mathrm{Vec}(X)`
+    (the vectorisation of :math:`X`) has a multivariate normal distribution
+    with mean :math:`\mathrm{Vec}(M)` and covariance :math:`V \otimes U`
+    (where :math:`\otimes` is the Kronecker product).
```

Comment:
Having this paragraph is better then not having it, but could you reword it a bit. Specifically, "vectorization" is heavily used for something different in numpy land (whether it's -s- or -z- :-). Would also be good to mention that this is equivalent but algorithmically inefficient.